audio/BRRTools/Portfile

```diff
@@ -0,0 +1,27 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        Optiroc BRRTools 3.14 v
+categories          audio
+maintainers         {gmail.com:chrisbodek @cbodek} openmaintainer
+license             permissive
+description         Tools to convert between .wav and SNES BRR files
+long_description    BRRTools are currently the most evolved tools to \
+                    convert between standard RIFF .wav format and SNES's \
+                    built-in BRR sound format.
+platforms           darwin
+
+checksums           rmd160  149ba55f45b996fb0b3548faeafa4b29ec02471c \
+                    sha256  7d09bcfc5b58491ea68700130349cc9583296847325680ece41d295d0860431f \
+                    size    14731
+                                        
+use_configure       no
```

Comment:
As @kencu was saying, you need to make sure you are [using the right compiler]. (https://trac.macports.org/wiki/UsingTheRightCompiler)